# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1273262007 -3600
#      Fri May 07 20:53:27 2010 +0100
# Node ID 09e244649f508e68d6764635a9a46a00ebb9b4b7
# Parent  31717cc7eeb398f4a8e6375da24a2a6bf01bff0a
Export symbol Vsparse_auto_mutate

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-05-07  Michael Goffioul  <michael.goffioul@gmail.com>
+
+	* ov-base.h (Vsparse_auto_mutate): Add OCTINTERP_API tag.
+
 2010-05-06  John W. Eaton  <jwe@octave.org>
 
 	* xgl2ps.c: Don't compile gl2ps.c unless we have OpenGL.
 	From Petr Mikulik <mikulik>.
 
 2010-05-06  John W. Eaton  <jwe@octave.org>
 
 	* ov-range.cc (octave_range::index_vector):
diff --git a/src/ov-base.h b/src/ov-base.h
--- a/src/ov-base.h
+++ b/src/ov-base.h
@@ -752,11 +752,11 @@ private:
   static int curr_print_indent_level;
   static bool beginning_of_line;
 
   DECLARE_OV_BASE_TYPEID_FUNCTIONS_AND_DATA
 };
 
 // TRUE means to perform automatic sparse to real mutation if there
 // is memory to be saved
-extern bool Vsparse_auto_mutate;
+extern OCTINTERP_API bool Vsparse_auto_mutate;
 
 #endif
